src/components/Admin/AdminTwitterAccounts.js

```diff
@@ -1,80 +1,42 @@
-import {DataGrid} from './DataGrid';
+import { DataGrid } from './DataGrid';
 import React from 'react';
 import Fluxxor from 'fluxxor';
 
-const FluxMixin = Fluxxor.FluxMixin(React),
-    StoreWatchMixin = Fluxxor.StoreWatchMixin("AdminStore");
-
-const columns = [
-    {editable: true, 
-     required: true,
-     key: "accountName", 
-     name: "Twitter App Name"},
-    {
-        editable:true,
-        sortable : true,
-        filterable: false,
-        resizable: true,
-        required: true,
-        name: "Consumer Key",
-        key: "consumerKey"
-    },
-    {
-        editable:true,
-        sortable : true,
-        required: true,
-        filterable: false,
-        resizable: true,
-        name: "Consumer Secret",
-        key: "consumerSecret"
-    },
-    {
-        editable:true,
-        sortable : true,
-        required: true,
-        filterable: false,
-        resizable: true,
-        name: "Token",
-        key: "token"
-    },
-    {
-        editable:true,
-        sortable : true,
-        required: true,
-        filterable: false,
-        resizable: true,
-        name: "Token Secret",
-        key: "tokenSecret"
-    }
-];
+const FluxMixin = Fluxxor.FluxMixin(React);
+const StoreWatchMixin = Fluxxor.StoreWatchMixin("AdminStore");
 
 export const AdminTwitterAccounts = React.createClass({
-    mixins: [FluxMixin, StoreWatchMixin],
+  mixins: [FluxMixin, StoreWatchMixin],
+
+  componentDidMount() {
+    this.getFlux().actions.ADMIN.load_twitter_accounts();
+  },
+
+  getStateFromFlux() {
+    return this.getFlux().store("AdminStore").getState();
+  },
+
+  handleSave(mutatedRows, columns) {
+    this.getFlux().actions.ADMIN.save_twitter_accts(this.props.siteKey, mutatedRows);
```

Comment:
Should this action also be renamed for consistency?

Reply:
oh yea I didn't see that thanks